### PR TITLE
Blog App: Create data model #1

### PR DESCRIPTION
**Pull request by siddghosh108**
- [ ] Create a new app.
- [ ]  Initialize the project with Git.
- [ ]  The project has Postgres database set up.
- [ ]  Build the project schema.

- Create and run the necessary migration files.
- Table and column names should match the ERD diagram.
-  note: photo for users table should be a link to an image
- Foreign keys should be included.
- Pay attention that in posts table the column holding the foreign key to the users table should be named author_id
- All columns that are foreign keys should have a corresponding index.

blog_app_erd_v1_1

### CODE REVIEW

File: db/migrate/20230926140450_create_posts.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
+class CreatePosts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.references :author, foreign_key: { to_table: 'users' }, null: false
+      t.string :title
+      t.text :text
+      t.integer :commentsCounter
+      t.integer :likesCounter
```
Comment (julie-ify) [inline db/migrate/20230926140450_create_posts.rb]: - [ ] Kindly follow the Ruby naming conventions. These should be `comments_counter` and `likes_counter`

File: README.md [hunk 2]
```diff
@@ -1 +1,140 @@
-# blog-app
\ No newline at end of file
+<div align="center">
```
Comment (julie-ify) [inline README.md:1]: - [ ] **[OPTIONAL]** You have over 3000+ files changed due to the `tmp` directory. You do not have a `.gitignore` file. Kindly create a `.gitignore` file and use the default Rails settings for it.

  ```
  # See https://help.github.com/articles/ignoring-files for more about ignoring files.
  #
  # If you find yourself ignoring temporary files generated by your text editor
  # or operating system, you probably want to add a global ignore instead:
  #   git config --global core.excludesfile '~/.gitignore_global'
  
  # Ignore bundler config.
  /.bundle
  
  # Ignore all logfiles and tempfiles.
  /log/*
  /tmp/*
  !/log/.keep
  !/tmp/.keep
  
  # Ignore pidfiles, but keep the directory.
  /tmp/pids/*
  !/tmp/pids/
  !/tmp/pids/.keep
  
  # Ignore uploaded files in development.
  /storage/*
  !/storage/.keep
  .byebug_history
  
  # Ignore master key for decrypting credentials and more.
  /config/master.key
  ```

**Review by julie-ify — CHANGES_REQUESTED** [review]
## Hi @siddghosh108,

Good job so far!

## Highlights!
- Good code organization and working app ✔️
- Descriptive README ✔️ 

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@julie-ify` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: db/migrate/20230926140450_create_posts.rb (hunk 1 above)
Comment (siddghosh108) [reply db/migrate/20230926140450_create_posts.rb]: Made the changes you asked for
class CreatePosts < ActiveRecord::Migration[7.0]
  def change
    create_table :posts do |t|
      t.references :author, foreign_key: { to_table: 'users' }, null: false
      t.string :title
      t.text :text
      t.integer :comments_counter
      t.integer :likes_counter 

      t.timestamps 
    end
  end
end

**Review by julie-ify — APPROVED** [review]
Hi @siddghosh108,

## Highlights!
- Good code organization and working app ✔️
- Descriptive README ✔️ 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Status: Approved ✔️ 

Cheers and Happy coding!👏👏👏

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- [ ] **[OPTIONAL]** Kindly add the `.gitignore` file I suggested earlier.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**